src/Field.js

```diff
@@ -54,8 +54,10 @@ const createField = ({ deepEqual, getIn }) => {
     }
 
     render() {
-      const { ConnectedField } = this
-      return <ConnectedField {...this.props} ref="connected"/>
+      return createElement(this.ConnectedField, {
+        ...this.props,
+        ref: 'connected'
+      })
```

Comment:
Anything in particular gained by avoiding JSX here? or just using the `createElement()` since we already have it in scope?